internal/examples/go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/stretchr/testify v1.4.0
 	go.uber.org/fx v1.10.0
 	go.uber.org/multierr v1.4.0
-	go.uber.org/thriftrw v1.22.0
+	go.uber.org/thriftrw v1.25.0
```

Comment:
Is it a mandatory change ? If no I would prefer to have this in another PR

Reply:
Detaching these changes from this pr :-)